cookbook/bundles/best_practices.rst

```diff
@@ -350,6 +350,13 @@ there are 3 modes, which the user can configure in their project:
 * 2.5-BC: the new 2.5 API with a backwards-compatible layer so that the
   2.4 API still works. This is only available in PHP 5.3.9+.
 
+.. note::
+
+    Starting with Symfony 2.7, the support for the 2.4 API has been
+    dropped and the minimal PHP version required for Symfony was
+    increased to 5.3.9. If your bundles requires Symfony >=2.7, you
+    don't need to take care about the 2.4 API anymore.
```

Comment:
@xabbuh my plan for Sunday is to drop the distinction between 2.5 and 2.5-BC, moving the BC layer in the 2.5 implementation itself (the only reason why we have separate implementations in 2.5 and 2.6 is because we decided to avoid bumping the Symfony requirement at this time and the BC layer cannot be implemented before 5.3.9. Bumping the requirement allows us to have a much better experience (allowing to have the new API always available while still having the BC layer all the time)